### DISCUSSION TITLE
这下载器是被空洞侵蚀了吗？你们都是这个速度吗？

### DISCUSSION THREAD

**Discussion by SednaMi** [share121/inter-knot#1768] [其他]
这破玩意，1MB/s。要下到什么时候啊啊啊啊啊啊啊啊啊啊
![image](https://github.com/user-attachments/assets/36936d27-ebed-4482-a544-b26269366763)
![image](https://github.com/user-attachments/assets/029d2a10-3e34-4c07-ad2a-952b9d26e56b)

**Comment by ZhuYuan-148** [comment]
是市民的网络问题吧，要不联系一下当地代理商？

**Reply by SednaMi** [reply]
下其他的都还好，就是这个启动器慢的要死。。。/(ㄒoㄒ)/~~

**Comment by hdaihdsu** [comment]
我的挺快的，我去刷了会视频就好了。

**Comment by 1451374815** [comment]
我没梯子，都下载不了，难受

**Comment by lamtt26** [comment]
不道啊，我比你快

**Comment by Rain-YYC** [comment]
我这100mb/s左右.....

**Comment by AmadeusDAZE** [comment]
建议新买个路由器......

**Comment by SWYH34827** [comment]
我的是挺快的，可能是你网不好🤷‍♂️

**Comment by createpinocao** [comment]
可以看看WIFI连接速度,正常情况下5g频段的连接速度大概在2400Mbps左右

**Comment by 1luik** [comment]
吧vpn关了试试

**Comment by blqdswkl** [comment]
我家网络是这样的

**Comment by RGthx** [comment]
考虑一下改改dns
看看是不是网线啥的问题
协商速率看看是不是千兆啥的

**Comment by yivitte** [comment]
应该是光纤或者传输的问题，如果你连得是无线就靠近一下光猫，如果你用的是以太网就试试把光纤升级到万兆，现在大三家的光纤都支持直接升级不需要换了

**Comment by fish-phaethon** [comment]
我曾经也是这样，网上任何方法都试过了还是不行，最后重启了一下电脑恢复了
